05week/checkers.js

```diff
@@ -50,20 +129,70 @@ function Board() {
       string += "\n";
     }
     console.log(string);
-  };
+  }
 
-  // Your code here
 }
-function Game() {
 
-  this.board = new Board();
+class Game {
+  constructor() {
+    this.board = new Board();
+
+    this.start = function() {
+      this.board.createGrid();
+      this.board.makeTheCheckers();
+    }
+  }
+
+
+
+  isValidMove(whatPiece, moveTo) {
```

Comment:
This function needs a return in order to complete your moveChecker function.  This is currently just reassigning checker positions